### PR TITLE
Reduced frame, remove testing logs

### PR DESCRIPTION
**Pull request by auriv**

### CODE REVIEW

File: vp8/encoder/onyx_if.c [hunk 1]
```diff
@@ -1273,14 +1273,22 @@ int vp8_reverse_trans(int x) {
 void vp8_new_framerate(VP8_COMP *cpi, double framerate) {
 /* Proximie */
 #if 1
+  if (framerate < .1) framerate = 24; /* Proximie uses 24 as default frame rate */
+ #if 0 /* For testing */
   if (framerate < .1) {
     static int px_cnt = 0;
```
Comment (richardscreene) [inline vp8/encoder/onyx_if.c:1279]: Isn't the following #if'd out?  So, why the changes?

**Review by richardscreene — APPROVED** [review]

File: vp8/encoder/onyx_if.c (hunk 1 above)
Comment (auriv) [reply vp8/encoder/onyx_if.c:1279]: was using these for debugging the issue facing with low fps/bitrate, potentially was thinking to eventually put these into production to help us troubleshoot. will wait to see any customers reporting similar issue i faced, we might need to enable this then.
